roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -95,6 +95,20 @@
     - port: 443
       targetPort: "oaproxy"
 
+- name: Set memcached service
```

Comment:
@ewolinetz, @jcantril, can you two explain more about your comments?  This is being added to the `openshift_logging_kibana` role right now.  Are we suggesting a new role, `openshift_logging_memcached`?  Or something else?  Can you provide some examples of what you two are thinking of?